src/app/players/players.component.ts

```diff
@@ -160,7 +169,7 @@ export class PlayersComponent implements OnInit, OnDestroy {
     this.playersService.removePlayer(player.name);
   }
 
-  trackByName(index, player: IPlayer) {
+  trackByName(player: IPlayer) {
```

Comment:
When using trackBy, the first argument is the index so that's a regression

Reply:
Updated.